lecli/query/commands.py

```diff
@@ -0,0 +1,141 @@
+@click.option('--dateto',
+              help='Date/Time to get events to (ISO-8601 datetime)')
+@click.option('-r', '--relative_range',
+              help='Relative range to query until now (Examples: today, yesterday, '
+                   'last x timeunit: last 2 hours, last 6 weeks etc.')
+def get_events(logkeys, lognick, loggroup, timefrom, timeto, datefrom, dateto, relative_range):
+    """Get log events"""
+
+    if lognick:
+        logkeys = api_utils.get_named_logkey(lognick)
+    elif loggroup:
+        logkeys = api_utils.get_named_logkey_group(loggroup)
+
+    if all([logkeys, timefrom, timeto]):
+        query_api.get_events(logkeys, time_from=timefrom, time_to=timeto)
+    elif all([logkeys, datefrom, dateto]):
+        query_api.get_events(logkeys, date_from=datefrom, date_to=dateto)
+    elif all([logkeys, relative_range]):
+        query_api.get_events(logkeys, time_range=relative_range)
+    else:
+        click.echo("Example usage: lecli get events 12345678-aaaa-bbbb-1234-1234cb123456 "
+                   "-f 1465370400 -t 1465370500")
+        click.echo("Example usage: lecli get events 12345678-aaaa-bbbb-1234-1234cb123456 "
+                   "--datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59' ")
+        click.echo("Example usage: lecli get events --loggroup myloggroup "
+                   "--datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59' ")
+        click.echo("Example usage: lecli get events --lognick mynicknamedlog "
+                   "--datefrom '2016-05-18 11:04:00' --dateto '2016-05-18 11:09:59' ")
+        click.echo("Example usage: lecli get events --lognick mynicknamedlog "
+                   "-r 'last 3 hours'")
+
+
+@click.command()
+@click.argument('logkeys', type=click.STRING, nargs=-1)
+@click.option('-n', '--lognick', default=None,
+              help='Nickname of log in config file')
+@click.option('-g', '--loggroup', default=None,
+              help='Name of log group defined in config file')
+@click.option('-l', '--last', default=1200,
+              help='Time window from now to now-X in seconds over which events will be returned '
+                   '(Defaults to 20 mins)')
+@click.option('-r', '--relative_range',
+              help='Relative range to query until now (Examples: today, yesterday, '
+                   'last x timeunit: last 2 hours, last 6 weeks etc.')
+def get_recent_events(logkeys, lognick, loggroup, last, relative_range):
+    """Get recent log events"""
+
+    if lognick:
+        logkeys = api_utils.get_named_logkey(lognick)
+    elif loggroup:
+        logkeys = api_utils.get_named_logkey_group(loggroup)
+
+    if all([logkeys, relative_range]):
+        query_api.get_recent_events(logkeys, time_range=relative_range)
+    elif all([logkeys, last]):
+        query_api.get_recent_events(logkeys, last_x_seconds=last)
+    else:
+        click.echo(
+            'Example usage: lecli recentevents \'12345678-aaaa-bbbb-1234-1234cb123456\' -l 200')
```

Comment:
i think this should be `lecli get recentevents` and i think we can get rid of escape character before and after logkey